docs/tutorials/intro.md

```diff
@@ -68,27 +71,28 @@ fn main() {
         .expect("Failed to parse locale.");
 
     if loc.id.language.as_str() == "es" {
-        println!("¡Hola amigo!");
+        println!("¡Hola!");
     }
 
     println!("You are using: {}", loc);
 }
 ```
-*Notice:* `ICU4X` canonicalized the locales's syntax which uses lowercase letter for the language portion.
+*Notice:* Here, `ICU4X` canonicalized the locales's syntax which uses lowercase letters for the language portion.
 
-After saving it, call `cargo run` in `~/projects/icu/myapp` and it should display:
+After saving it, call `cargo run` in `~/projects/icu_tutorial/myapp` and it should display:
 
 ```
-¡Hola amigo!
+¡Hola!
 You are using: es-AR
 ```
 
-Congratulations! `ICU4X` has been used to semantically operate on a locale and the first string is now displayed only if the user is using a locale with Spanish `language` part!
+Congratulations! `ICU4X` has been used to semantically operate on a locale!
 
 ## Convenience macro
```

Comment:
Nit: This doesn't seem important enough to warrant its own heading. Consider just mixing it in as an example in the above section.